task5/src/main/java/com/kwitter/GetServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.kwitter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/get")
+public class GetServlet extends HttpServlet {
+
+    final String lengthLimitExceededText = "length of name more than 100 symbols";
+    final int maxLength = 100;
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
+        String name = req.getParameter("name");
+        PrintWriter stream = resp.getWriter();
+        if (name != null) {
```

Comment:
Please avoid nested code block wherever possible, and specifically nested conditions. The good point is to use early fail / early return approach
```
if (name == null) {
    return;
}
if (name.length ... )
```